src/package_manifest.rs

```diff
@@ -45,17 +46,34 @@ impl DependencyGroup {
 impl ConfigurationFile<PackageManifest> for PackageManifest {
     const FILENAME: &'static str = "package.json";
 
-    fn from_directory<P>(monorepo_root: P, directory: P) -> Result<PackageManifest>
-    where
-        P: AsRef<Path>,
-    {
-        let containing_directory = monorepo_root.as_ref().join(&directory);
-        let file = File::open(containing_directory.join(Self::FILENAME))?;
-        let reader = BufReader::new(file);
-        let manifest_contents: PackageManifestFile = serde_json::from_reader(reader)?;
+    fn from_directory(monorepo_root: &Path, directory: &Path) -> Result<PackageManifest> {
+        let filename = monorepo_root.join(directory).join(Self::FILENAME);
+        let manifest_contents: PackageManifestFile =
+            read_json_from_file(&filename).with_context(|| {
+                formatdoc!(
+                    "
+                    Unexpected contents in {:?}
+
+                    I'm trying to parse the following properties and values out
+                    of this package.json file:
+
+                    - name: string
+                    - version: string
+
+                    and if the any of the following values are present, I expect
```

Comment:
@EricCrosson extraneous `the` here

Reply:
Hot dog @louib-bitgo you are on fire catching these typos! :hotdog: :hotdog: :hotdog:

Reply:
Addressed in #192